expected/pathman_permissions.out

```diff
@@ -12,15 +12,23 @@ CREATE TABLE permissions.user1_table(id serial, a int);
 INSERT INTO permissions.user1_table SELECT g, g FROM generate_series(1, 20) as g;
 /* Should fail (can't SELECT) */
 SET ROLE user2;
-SELECT create_range_partitions('permissions.user1_table', 'id', 1, 10, 2);
-ERROR:  permission denied for relation user1_table
+DO $$
+BEGIN
+    SELECT create_range_partitions('permissions.user1_table', 'id', 1, 10, 2);
+EXCEPTION
+    WHEN insufficient_privilege THEN
```

Comment:
Shouldn't we raise a notice in case of `insufficient_privilege`? Otherwise we won't know if this call failed.

Reply:
Well, if it is indeed `insufficient_privilige`, then that's what we are after here, so silently eating it is fine. Actually, an attempt to *hide* the error is the only reason for this change -- one word changed in the error message from `relation` to `table` in 10 and 11 PG. If this is any other exception, it will fall through shouting along the way.

Reply:
(We still can add some unifying NOTICE in exception handler or just a comment explaining this, if you like)

Reply:
Fixed